frontend/src/metabase/nav/components/ReadOnlyBanner/ReadOnlyBanner.tsx

```diff
@@ -5,8 +5,8 @@ import { Flex, Icon, Text } from "metabase/ui";
 export const ReadOnlyBanner = () => {
   return (
     <Flex py="0.75rem" px="1rem" bg="accent4" gap="md" align="center">
-      <Icon color="black" name="info_filled" />
-      <Text fw="bold" color="black">
+      <Icon color="text-dark" name="info_filled" />
+      <Text fw="bold" color="text-dark">
```

Comment:
Black was `#2E353B` while text-dark is `#4C5773`, I think it's a slightly different shade?